docs/get-started/web.md

```diff
@@ -0,0 +1,106 @@
+---
+title: In the browser
+---
+
+# Get started in the browser 
+
+The fastest and easiest way to create a Pack is to use our web-based editor. You don't need to download any code or tools, just make sure you have:
+
+1. A Coda account, with
+2. [Doc Maker access](https://help.coda.io/en/articles/3388781-members-and-roles) in your workspace.
+
+If you're new to Coda, [sign up](https://{{coda.domain}}/signup) for a free account and you'll automatically be made a Doc Maker in your personal workspace.
+
+## Navigate to the Pack editor
```

Comment:
need to confirm with @punit-codaio - but we may need to swap mentions of 'Pack editor' for 'Pack studio' (or be careful about only using the word "editor" to refer to the actual IDE/code editor part of the UI)

Reply:
As per the Slack thread, I've updated this to "Pack Studio".